Packs/CommonScripts/TestPlaybooks/playbook-FailedInstances_-_Test.yml

```diff
@@ -0,0 +1,237 @@
+      version: -1
+      name: Check for Failed Instances
+      description: Executes test for all integration instances available and returns
+        a detailed table about failed integration instances
+      scriptName: FailedInstances
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "3"
+      - "5"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 420
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "3":
+    id: "3"
+    taskid: 7b44dafc-e8f6-4536-8510-72270814e314
+    type: condition
+    task:
+      id: 7b44dafc-e8f6-4536-8510-72270814e314
+      version: -1
+      name: Check good instance
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "yes":
+      - "4"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: FailedInstances.brand
+            iscontext: true
+          right:
+            value:
+              simple: Carbon Black Enterprise EDR
+      - - operator: isEqualString
+          left:
+            value:
+              simple: FailedInstances.information
+            iscontext: true
+          right:
+            value:
+              simple: succeed
+      - - operator: isEqualString
+          left:
+            value:
+              simple: FailedInstances.status
+            iscontext: true
+          right:
+            value:
+              simple: good
+    view: |-
+      {
+        "position": {
+          "x": 730,
+          "y": 590
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "4":
+    id: "4"
+    taskid: ebcbb903-e2b8-454d-8e39-8323b0d31e3e
+    type: title
+    task:
+      id: ebcbb903-e2b8-454d-8e39-8323b0d31e3e
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 870
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "5":
+    id: "5"
+    taskid: aab31f3e-8ea4-4d46-88c7-cd3e762f1a0b
+    type: condition
+    task:
+      id: aab31f3e-8ea4-4d46-88c7-cd3e762f1a0b
+      version: -1
+      name: Check failed instance
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "yes":
+      - "4"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: FailedInstances.brand
+            iscontext: true
+          right:
+            value:
+              simple: fcm
```

Comment:
what is `fcm`?

Reply:
I'm not really sure, but its instance failed on the build, which is good for the test